src/core/client/admin/routes/Configure/sections/General/FlairBadgeImagePreview.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FunctionComponent, useCallback, useState } from "react";
+
+import { SvgIcon } from "coral-ui/components/icons";
+import ImageFileWarningIcon from "coral-ui/components/icons/ImageFileWarningIcon";
```

Comment:
this can just be imported from `coral-ui/components/icons` along with the `SvgIcon` now!

Reply:
fixed